views/js/runner/plugins/security/fullScreen.js

```diff
@@ -250,13 +258,35 @@ define([
                 testRunner.trigger('disablenav disabletools');
             }
 
+            function handleResizeToFullScreenChange() {
```

Comment:
**question:** Why is it needed to add this function? We already have the `handleFullScreenChange()`, [line 318](https://github.com/oat-sa/extension-tao-test-runner-plugins/blob/21c6b6d5c82eb2ccd694c2c58e3ca0390e8e9276/views/js/runner/plugins/security/fullScreen.js#L318). The only missing part is the dialog closure when the fullscreen is set: `testRunner.trigger('closedialog.fullscreen');`

**suggestion:** Enhance the existing handler
```javascript
            function handleFullScreenChange() {
                isFullScreen = checkFullScreen();
                if (!isFullScreen) {
                    leaveFullScreen(testRunner);
                    alertUser();
                } else {
                    enterFullScreen(testRunner);
                    testRunner.trigger('closedialog.fullscreen');
                }
            }
```